lib/metanorma/cli/commands/config.rb

```diff
@@ -88,7 +92,8 @@ def load_config(global_config, create_default_config: true)
 
           save_default_config(config_path)
 
-          [::YAML::load_file(config_path).symbolize_all_keys || {}, config_path]
+          [::YAML::load_file(config_path).symbolize_all_keys || {},
```

Comment:
Is this something we can clean up with using the `SiteManifest`? We don't want direct YAML loads. Thanks.

Reply:
I'll see if we can use `SiteManifest` to load it instead.

Reply:
This config refers to `~/.metanorma/config.yml` so it's not related to `SiteManifest`.

We can model that in a separate PR.

Reply:
Created #365

Reply:
Got it, thanks!